packages/@aws-cdk/assertions/vendor-in.sh

```diff
@@ -21,6 +21,15 @@ set -euo pipefail
 dest="lib/vendored"
 mkdir -p $dest
 
+if [[ $scriptdir != *"/packages/@aws-cdk/assertions"* ]]; then
+  # Make the script "fail closed" when running in the @aws-cdk-lib build context.
```

Comment:
Here's a 💯 nitpick, but this is not what "fail closed" means. "Fail closed" means to NOT proceed with something if you're unsure about the current state of the system. I think this is more akin to "fail open", proceed regardless.

And it's something we probably shouldn't be doing either way :)

Reply:
Good callout. I changed it to just short-circuit instead.